superset/cli.py

```diff
@@ -109,7 +109,10 @@ def version(verbose: bool) -> None:
 
 
 def load_examples_run(
-    load_test_data: bool, only_metadata: bool = False, force: bool = False
+    load_test_data: bool,
+    load_big_data: bool,
+    only_metadata: bool = False,
+    force: bool = False,
```

Comment:
this is a bit of a nit, but for consistency, do you think it's better to have default `False` values on all of these or allow nulls?

Reply:
In this case `click` is adding the default values when parsing the CLI options, but I'll add some default values to the function signature as well.